### DISCUSSION TITLE
A question & images regarding iCub skin

### DISCUSSION THREAD

**Discussion by frank-foerster** [robotology/community#110] [Q&A] [answered]
Dear all,

I have been contacted by a content developer from the Science museum in London with regards to the iCub skin. As I know next to nothing about its skin, I decided to ask the wider iCub community, that is you, for help. 
1. Firstly, she wants to know whether the patch of skin on the attached image is identical to the one employed on the iCub. 
   
   So my **first question**: is the skin used on the iCub identical to the one visible on the attached image? (This is assuming that one can tell by mere visual inspection. If this is impossible to tell, this information might also be helpful for them.)
2. Secondly, she was asking whether there were any images of a _stripped down_ iCub, that is, an iCub where its skin sensors are exposed. 
   They would like to use such an image in their upcoming robot exhibition. The purpose of this image would be to give visitors a better idea of how such skin is employed on an actual robot, and what the difficulties might be, such as potential problems with pliability if one were to add skin on the _elbow_ or other more _bendy_ parts of a robot etc. 
   
   So my **second question**: does anyone have images of an _exposed_ iCub, optimally in high quality, which they would be willing to _borrow_ to the museum?
   If not, I will do the _stripping_ myself, but if anyone already has such images, it would save me some time.

Thanks.

![icub_skin](https://cloud.githubusercontent.com/assets/16962752/14920026/64327422-0e23-11e6-8633-facb30f7f355.jpg)

**Comment by pattacini** [comment]
Maybe @maggia80 can help in this respect.

**Comment by maggia80** [comment, marked as answer]
Hi Frank,

No, this is not the iCub skin. I would like to know where she found it, if it is possible. Here in attachment the picture of the icub skin. I'll check if I can find also a picture of the skin on iCub.

![skintriangles](https://cloud.githubusercontent.com/assets/6987330/14948932/d65755c6-1045-11e6-9544-6c5be4d27d99.png)

**Comment by maggia80** [comment]
A few more pictures:

https://www.dropbox.com/sh/qf4bb19xha1u0w9/AAB6O6ElGjWi2BemU8adXhaXa?dl=0

**Comment by frank-foerster** [comment]
Thanks for the images. I'll forward them to Anna. If they like them, is it ok for them to use them in their exhibition?

I will also tell her that the two types of skin are not the same, ask her for some more detail, and will get back to you.

**Comment by pattacini** [comment]
Hi @frank-foerster 

There's no problem in using the pictures in the exhibition.

**Comment by alecive** [comment]
@matejhof should have a pretty good picture of the blue robot with all the skin parts exposed if I recall correctly!

**Comment by frank-foerster** [comment]
Hi @maggia80, Anna finally got back to me. Evidently the piece of skin on the image is a loan given to them by Giorgio Cannata at the University of Genoa who were involved Roboskin (and are presumably around your corner). She sent me a link to a leaflet from Roboskin on which there are images of similar skin as the one on the image she sent. 
(http://cordis.europa.eu/docs/projects/cnect/0/231500/080/publishing/readmore/ROBOSKIN-Leaflet.pdf)
Maybe there has been some early version prior to the one used now? 
Anyways, she was asking me whether she can contact you directly as they would need higher-resolution images if available. Can I forward you email address to her? (As I don't have yours you'd have to send that one to me, my email address is on my github profile page).
@alecive: Thanks for the hint. I'll try to contact Matej directly. 

Thanks for all the help.

**Comment by maggia80** [comment]
ok yes I realized that it could be a picture from Giorgio. give my contact directly to her

**Comment by frank-foerster** [comment]
@maggia80: I actually don't have your email address (and it's not stated on your Github profile). 
Could you send it to me? (I put mine on the profile).
Thanks

**Comment by maggia80** [comment]
I sent you my  mail

**Comment by pattacini** [comment]
Closing here.
Feel free to reopen.
